### DISCUSSION TITLE
How to install extensions

### DISCUSSION THREAD

**Discussion by shlingok** [pimcore/pimcore#10875] [General]
Using command , I got no results,  such as :
root@1e9e35dc82bf:/var/www/html/pimcore# composer search pimcore/data-hub-simple-rest
root@1e9e35dc82bf:/var/www/html/pimcore#
Then, I seach in https://packagist.org/，There was also no result：
![image](https://user-images.githubusercontent.com/6915649/142837747-31f3a61d-c288-472f-ac19-72b27d571e4f.png)

**Comment by ckemptner** [comment]
Hi @shlingok! This a private package, which can only be obtained for active customers of the Pimcore Enterprise Edition. You can learn more about our Editions here https://pimcore.com/editions Please contact our sales team to know more!

**Reply by shlingok** [reply]
OK, 3q
